vpr/src/base/read_circuit.cpp

```diff
@@ -36,7 +41,7 @@ AtomNetlist read_and_process_circuit(e_circuit_format circuit_format, t_vpr_setu
     bool verbosity = vpr_setup.NetlistOpts.netlist_verbosity;
 
     if (circuit_format == e_circuit_format::AUTO) {
-        auto name_ext = vtr::split_ext(circuit_file);
+        auto name_ext = vtr::split_ext(rd_circuit_file);
 
         VTR_LOGV(verbosity, "Circuit file: %s\n", circuit_file);
```

Comment:
Undefined variable circuit_file

Reply:
Ehhh, that's how solving rebase conflicts blindly ends... Should be fixed now.